### DISCUSSION TITLE
Hide some fields from list view, but not the details view

### DISCUSSION THREAD

**Discussion by dougw-bc** [tfwright/live_admin#81] [Q&A]
Some fields - mostly embedded structs are a bit of a nuisance on the list view - but still want visible on the details view + be editable.

Is there a way to do that?

`hidden_fields` seems to hide the fields from both list + details view.

**Comment by tfwright** [comment]
There is no config for it, but you could probably achieve it with CSS overrides using something like https://developer.mozilla.org/en-US/docs/Web/CSS/:nth-child

**Reply by Rio517** [reply]
Unfortunately, using CSS overrides like this is really brittle. Perhaps you could add field names to the CSS. Still, far from an ideal solution, but at least not so brittle.

**Reply by tfwright** [reply]
I would be open to a PR that adds the field name to the th and td elements on the list and form fields. Perhaps as a data rather than a CSS class to keep the latter as semantic as possible?
